src/client/keycode.h

```diff
@@ -44,6 +44,8 @@ class KeyPress
 	const char *sym() const;
 	const char *name() const;
 
+	irr::EKEY_CODE keycode() const { return Key; }
```

Comment:
perhaps std::hash can be declared a friend class instead of making this public, not sure